extensions/linz/examples/collection.json

```diff
@@ -12,6 +12,16 @@
   "title": "A title",
   "description": "A description",
   "license": "Apache-2.0",
+  "linz:asset_summaries": {
+    "created": {
+      "minimum": "1999-01-01T00:00:00Z",
```

Comment:
For us these examples will have an awkward-looking offset (ref e.g. scanning extension item example), not sure how authentic you want to be!

Reply:
I don't understand. Do you mean how the time is not New Zealand midnight? If so, why would it be?

Reply:
No the time is New Zealand midnight. `Z` denotes `UTC` so if we are representing a date without a known time then when we convert to `UTC` it'll be offset by 12 or 13 hours depending on whether it was during daylight saving. Then when clients change `UTC` to `NZT`, everything will look correct...

Reply:
I don't understand why we're focusing on NZ midnight though. Firstly it's just an example, and second it's not meant to be NZ-specific…

Reply:
That is a fair call for `asset_summaries`.